open-craft/opencraft
# [SE-2892] Fix issues with instance reference admin form and creator assignment

This PR:
- changes how the inlines instance references of the OpenedX instance are displayed on the admin page. Now you should see them expanded.
- allows you to save an instance without providing and `owner` or `creator`. By default, it sets these fields to the currently logged user.

**JIRA ticket**: [SE-2892](https://tasks.opencraft.com/browse/SE-2892)

**Testing**: 
- Go to an OpenedX instance on the admin page, edit it and verify that the instance reference at the bottom of the form is expanded
- Save a new Instance without providing an owner or created. You should be able to do it.

**Reviewers**
